docs/source/cli.rst

```diff
@@ -7,50 +9,53 @@ The examples below use the image at http://idr.openmicroscopy.org/webclient/?sho
 
 All examples can be made more or less verbose by passing `-v` or `-q` one or more times::
 
-    # ome_zarr -vvv ...
+    ome_zarr -vvv ...
 
 
 info
 ====
 
-Use the `ome_zarr` command to interrogate Zarr datasets::
+Use the `ome_zarr` command to interrogate Zarr datasets.
+
+Remote data::
 
-    # Remote data
-    $ ome_zarr info https://uk1s3.embassy.ebi.ac.uk/idr/zarr/v0.1/6001240.zarr/
+    ome_zarr info https://uk1s3.embassy.ebi.ac.uk/idr/zarr/v0.1/6001240.zarr/
 
-    # Local data (after downloading as below)
-    $ ome_zarr info 6001240.zarr/
+Local data::
+
+    ome_zarr info 6001240.zarr/
 
 view
 ====
 
 Use the `ome_zarr` command to view Zarr data in the https://ome.github.io/ome-ngff-validator::
 
-    # Local data (after downloading as below)
-    $ ome_zarr view 6001240.zarr/
+    ome_zarr view 6001240.zarr/
 
 download
 ========
 
 To download all the resolutions and metadata for an image::
 
-    # creates local 6001240.zarr/
```

Comment:
I think this line is still useful to keep, maybe add to the sentence above?

Reply:
👍 done